src/main/resources/sql_scripts/1base.sql

```diff
@@ -155,10 +155,11 @@ ALTER TABLE category_to_event
             REFERENCES event (id);
 
 
+
 INSERT INTO role ("name") VALUES

```

Comment:
to też powinno sie znaleźć w nowym pliku. Albo jako usunięcie wszystkiego z tej tabeli, albo jako update istniejących rekordów.

Co więcej te dane powinny być w pliku `static_data.sql`. 3 podstawowe pliki:
- 1_structure.sql (tylko tabele i constrainty)
- 2_static_data.sql (tylko dane, które są potrzebne do prawidłowego działania aplikacji)
- 3_test_data.sql (dane, które nie powinny trafić na produkcję - różne środowiska powinny uruchamiać różne zestawy plików)